aspnetcore/tutorials/first-mvc-app-xplat/new-field.md

```diff
@@ -0,0 +1,71 @@
+uid: tutorials/first-mvc-app-xplat/new-field
+---
+# Adding a new field
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+This tutorial will add a new field to the `Movies` table. We'll drop the database and create a new one when we change the schema (add a new field). This workflow works well early in development when we don't have any production data to perserve.
+
+Once your app is deployed and you have data that you need to perserve, you can't drop your DB when you need to change the schema. Entity Framework [Code First Migrations](http://docs.efproject.net/en/latest/platforms/aspnetcore/new-db.html) allows you to update your schema and migrate the database without losing data. Migrations is a popular feature when using SQL Server, but SQLlite does not support many migration schema operations, so only very simply migrations are possible. See [SQLite Limitations](https://docs.microsoft.com/ef/core/providers/sqlite/limitations) for more information.
+
+## Adding a Rating Property to the Movie Model
+
+Open the *Models/Movie.cs* file and add a `Rating` property:
+
+[!code-csharp[Main](../first-mvc-app/start-mvc/sample/MvcMovie/Models/MovieDateRating.cs?highlight=11&range=7-18)]
+
+Because you've added a new field to the `Movie` class, you also need to update the binding white list so this new property will be included. In *MoviesController.cs*, update the `[Bind]` attribute for both the `Create` and `Edit` action methods to include the `Rating` property:
+
+```csharp
+[Bind("ID,Title,ReleaseDate,Genre,Price,Rating")]
+   ```
+
+You also need to update the view templates in order to display, create and edit the new `Rating` property in the browser view.
+
+Edit the */Views/Movies/Index.cshtml* file and add a `Rating` field:
+
+[!code-HTML[Main](../first-mvc-app/start-mvc/sample/MvcMovie/Views/Movies/IndexGenreRating.cshtml?highlight=17,39&range=24-64)]
+
+Update the */Views/Movies/Create.cshtml* with a `Rating` field.
+
+The app won't work until we update the DB to include the new field. If you run it now, you'll get the following `SqliteException`:
+
+`SqliteException: SQLite Error 1: 'no such column: m.Rating'.``
+
+You're seeing this error because the updated Movie model class is different than the schema of the Movie table of the existing database. (There's no `Rating` column in the database table.)
+
+There are a few approaches to resolving the error:
+
+1. Drop the database and have the Entity Framework automatically re-create the database based on the new model class schema. This approach is very convenient early in the development cycle when you are doing active development on a test database; it allows you to quickly evolve the model and database schema together. The downside, though, is that you lose existing data in the database — so you don't want to use this approach on a production database! Using an initializer to automatically seed a database with test data is often a productive way to develop an app.
```

Comment:
The explanation here ("This approach is very convenient ..." ) is the same as what you have in the intro at top of page; I don't think you need it in both places.